subsys/shell/shell_cmds.c

```diff
@@ -422,7 +428,9 @@ SHELL_STATIC_SUBCMD_SET_CREATE(m_sub_backspace_mode,
 SHELL_STATIC_SUBCMD_SET_CREATE(m_sub_shell,
 	SHELL_CMD(backspace_mode, &m_sub_backspace_mode,
 			SHELL_HELP_BACKSPACE_MODE, NULL),
+#ifdef CONFIG_SHELL_VT100_COMMANDS
```

Comment:
Would you mind replace `ifdef` with: `SHELL_COND_CMD`?

Reply:
Sure, I've made the same for the clear function below.